index.md

```diff
@@ -35,7 +35,7 @@ Why should you use MeiliSearch instead of any other existing solution? We try to
 - **Instant Search** (answers < 50 milliseconds): Priority is given to fast answers for smooth search experience.
 - **Search as you type** (_prefix search_): Results are updated on each keystroke. To make this possible, we use a [prefix-search](/guides/advanced_guides/prefix.md#prefix-search).
 - [Typo tolerance](/guides/advanced_guides/typotolerance.md#typo-tolerance): Understands typo and miss-spelling.
-- [Tokenization](/guides/advanced_guides/tokenization.md) in **English**, **Chinese**, **Japanese**, **Korean**, and languages that use the Latin alphabet.
+- [Tokenization](/guides/advanced_guides/tokenization.md) in **English**, **Chinese**, and **all languages that separate words with whitespace**.
```

Comment:
Apparently the better word for this is `word divider` as per wikipedia: https://en.wikipedia.org/wiki/Word_divider#Types

```suggestion
- [Tokenization](/guides/advanced_guides/tokenization.md) in **English**, **Chinese**, and **all languages that uses space as a word divider**.
```
I'm really not sure of my suggestion so feel free to ignore. In which case LGTM 🔥